From 8334809e79c87f4a81ad8dd5c828369dde349a25 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Thu, 08 May 2025 20:31:14 +0300
Subject: [PATCH] [SW-227605] g2e: support bis-e pll speed

There is no change in the PLL speed, but still there is to support
a new entry for bis-e.

Change-Id: I94a4c1f3b44ab161b145e8a5d4f270eaf3d58741
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/arch/gaudi2/common.c b/arch/gaudi2/common.c
index cbd21c0..b1ec8bf 100644
--- a/arch/gaudi2/common.c
+++ b/arch/gaudi2/common.c
@@ -765,11 +765,13 @@
 		[GAUDI2_BIS_NONE] = GAUDI2_HIGH_SPEED_PLL,
 		[GAUDI2_BIS_REV_C] = GAUDI2_BIS_C_SPEED_PLL,
 		[GAUDI2_BIS_REV_D] = GAUDI2_BIS_D_SPEED_PLL,
+		[GAUDI2_BIS_REV_E] = GAUDI2_BIS_E_SPEED_PLL,
 	};
 	uint32_t default_id_map[GAUDI2_BIS_REV_MAX] = {
 		[GAUDI2_BIS_NONE] = GAUDI2_PLL_TABLE_DEFAULT,
 		[GAUDI2_BIS_REV_C] = GAUDI2C_PLL_TABLE_DEFAULT,
 		[GAUDI2_BIS_REV_D] = GAUDI2D_PLL_TABLE_DEFAULT,
+		[GAUDI2_BIS_REV_E] = GAUDI2E_PLL_TABLE_DEFAULT,
 	};
 
 	/* MSS pll is selected according to HBM pll */
diff --git a/include/arch/gaudi2/common.h b/include/arch/gaudi2/common.h
index a51a0ea..b585b7d 100644
--- a/include/arch/gaudi2/common.h
+++ b/include/arch/gaudi2/common.h
@@ -114,9 +114,11 @@
 #define GAUDI2_BIS_C_SPEED_PLL			1095 /* MHz */
 #define GAUDI2_BIS_D_OBSOLETE_SPEED_PLL		483  /* MHz */
 #define GAUDI2_BIS_D_SPEED_PLL			1650 /* MHz */
+#define GAUDI2_BIS_E_SPEED_PLL			1650 /* MHz */
 #define GAUDI2_PLL_TABLE_DEFAULT		1 /* table1 (1650MHz) */
 #define GAUDI2C_PLL_TABLE_DEFAULT		2 /* table2 (1095Mhz MME) */
 #define GAUDI2D_PLL_TABLE_DEFAULT		1 /* table1 (1650Mhz MME) */
+#define GAUDI2E_PLL_TABLE_DEFAULT		1 /* table1 (1650MHz MME) */
 #define EEPROM_BUF_SZ				DT_PROP(DT_NODELABEL(eeprom), \
 						size)
 
